backend/src/database/repositories/organizationRepository.ts

```diff
@@ -1617,12 +1619,8 @@ class OrganizationRepository {
       FROM organizations o
       ${
         withAggregates
-          ? ` INNER JOIN "organizationSegmentsAgg" osa ON osa."organizationId" = o.id AND ${
-              segmentId ? `osa."segmentId" = $(segmentId)` : `osa."segmentId" IS NULL`
-            }`
-          : ` LEFT JOIN "organizationSegmentsAgg" osa ON osa."organizationId" = o.id AND ${
-              segmentId ? `osa."segmentId" = $(segmentId)` : `osa."segmentId" IS NULL`
-            }`
+          ? ` INNER JOIN "organizationSegmentsAgg" osa ON osa."organizationId" = o.id AND osa."segmentId" IN ($(segments:csv))`
```

Comment:
I don't think this is going to work - We should still send a single segmentId to this query because we keep each subprojects, projects and project group aggregates in a single row in `organizationSegmentsAgg`